src/appleseed/renderer/kernel/lighting/drt/drtlightingengine.cpp

```diff
@@ -460,4 +461,36 @@ ILightingEngine* DRTLightingEngineFactory::create()
     return new DRTLightingEngine(m_light_sampler, m_params);
 }
 
+Dictionary DRTLightingEngineFactory::get_params_metadata()
+{
+    Dictionary metadata;
+    add_common_params_metadata(metadata);
+
+    metadata.dictionaries().insert(
+        "max_path_lenght",
```

Comment:
Typo: `max_path_length`

My advice: search & fix `lenght` everywhere in appleseed...